apps/admin-x-activitypub/src/components/layout/Onboarding/Step3.tsx

```
@@ -0,0 +1,371 @@
+                </div>
+            </div>
+            <div className='grid h-full grid-cols-[auto_248px] gap-4'>
+                <div className='mx-auto flex w-full max-w-[524px] flex-col items-center gap-4 pt-8'>
+                    <div className='mb-5 flex w-full items-center gap-3 rounded-md bg-white p-3 text-gray-600 shadow-sm'>
+                        <div className='h-9 w-9 rounded-full bg-gray-100'></div>
+                        What&apos;s new?
+                    </div>
+                    {feedList.map(item => (
+                        <>
+                            <div className='flex w-full items-start gap-3 px-3'>
+                                <div className='h-9 max-h-9 min-h-9 w-9 min-w-9 max-w-9 rounded-full bg-cover bg-center bg-no-repeat'
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)